action/QuadMove.action

```diff
@@ -10,6 +10,7 @@ float64 y_position
 float64 z_position
 float64 x_velocity
 float64 y_velocity
+float64 ending_radius_to_roomba # How close to get to the roomba for go_to_roomba
```

Comment:
Not sure what effects the comment makes here for the move actions, but you don't need it.

Reply:
Fixed. Where is the place for explaining these variables?

Reply:
Actually, you can move the comment to above the variable. so on line 13